docs/reference/migration/migrate_7_0/analysis.asciidoc

```diff
@@ -6,3 +6,10 @@
 The `delimited_payload_filter` is renamed to `delimited_payload`, the old name is 
 deprecated and will be removed at some point, so it should be replaced by 
 `delimited_payload`.
+
+
+==== Limiting the number of tokens produced by _analyze
+
+To safeguard against out of memory errors, the number of tokens that can be produced
+using `_analyze` endpoint has been limited to 10000. This default limit can be changed for
```

Comment:
nit: I think it should be "using the `_analyze` endpoint", but then again I'm not a native speaker. Feel free to change as you like or not, no extra review needed if you want to change anything here.

Reply:
thanks Christoph, will modify accordingly